src/types/dc_msg.ts

```diff
@@ -17,6 +18,21 @@ export type DCMessageJitter = number;
 export type TrackInfo = {
     type: string;
     sender_id: string;
+    mime_type: CodecMimeType;
```

Comment:
This change is crucial. It lets us clearly know the encoding of a remote track, which is needed to switch the receiving side from one track to the other, based on which one may have resumed sending, since we don't negotiate new tracks past the first iteration (either AV1->VP8 or VP8->AV1).